DESCRIPTION

```diff
@@ -0,0 +1 @@
+Config/renv/profiles/reporting/dependencies: quarto, leaflet, plotly, sf
```

Comment:
Adding a `DESCRIPTION` file with dependencies used by the `reporting` profile is a necessary step in order to be able to use a [profile](https://rstudio.github.io/renv/articles/profiles.html#managing-profile-specific-dependencies) in order to isolate reporting dependencies. More details on how this works in the README diff below.

Note that aside from `quarto`, these packages are not semantically important; I just chose a few packages that are used in `model_qc.qmd` at random for testing purposes.